### PR TITLE
Update: Glodon.CADReader.CN version 5.19.0.91

### PR DESCRIPTION
**Pull request by SpecterShell**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/116571&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/g/Glodon/CADReader/CN/5.19.0.91/Glodon.CADReader.CN.installer.yaml [hunk 1]
```diff
@@ -11,14 +11,13 @@ FileExtensions:
...
 - octet-stream
-ProductCode: '{DDD659B5-6B07-4F5A-A0D3-9E8D3147E165}_is1'
-ReleaseDate: 2023-04-20
 AppsAndFeaturesEntries:
 - DisplayName: CAD快速看图v5.19.0.91
   ProductCode: '{DDD659B5-6B07-4F5A-A0D3-9E8D3147E165}_is1'
 Installers:
 - Architecture: x86
-  InstallerUrl: https://yuntu-f.glodon.com/app/CADReaderPc.exe
+  InstallerUrl: https://yuntu-f.kuaicad.com/app/CADReaderPc_5.19.0.91.exe
```
[the remark this answers is not in the document]
Comment (SpecterShell) [reply manifests/g/Glodon/CADReader/CN/5.19.0.91/Glodon.CADReader.CN.installer.yaml:19]: The update API: https://cad.glodon.com/update/version/info/cadpc

[the remark this answers is not in the document]
Comment (SpecterShell) [reply manifests/g/Glodon/CADReader/CN/5.19.0.91/Glodon.CADReader.CN.installer.yaml:19]: @stephengillie

[the remark this answers is not in the document]
Comment (stephengillie) [reply manifests/g/Glodon/CADReader/CN/5.19.0.91/Glodon.CADReader.CN.installer.yaml:19]: Hi @SpecterShell,

Sorry for the delay. Can we replace the `PackageUrl` with the update API link you provided above? I'd suggest this but GitHub won't allow this for some reason.

[the remark this answers is not in the document]
Comment (SpecterShell) [reply manifests/g/Glodon/CADReader/CN/5.19.0.91/Glodon.CADReader.CN.installer.yaml:19]: @stephengillie 
Similar to https://github.com/microsoft/winget-pkgs/pull/116569

[the remark this answers is not in the document]
Comment (stephengillie) [reply manifests/g/Glodon/CADReader/CN/5.19.0.91/Glodon.CADReader.CN.installer.yaml:19]: Yes - I knew there were 2, but had lost this one. Thanks for helping it to the surface.

**Review by Trenly — APPROVED** [review]

**Pull request merged**